docs/apm/traces/working-with-tracing-data/service-map.md

```diff
@@ -1,64 +1,96 @@
 ---
 id: service-map
-title: Service Map
-sidebar_label: Service Map
+title: Services List and Map
 description: View your application topology and investigate microservice interactions.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Service Map is a high-level view of your application environment, automatically built from tracing data in real time as it arrives in Sumo Logic. This gives you a greater understanding of your application architecture and dependencies between monitored microservices.
+To open the Services List and Map, click **+ New** > **Services**.
 
-To open Service Map, click **+ New** > **Service Map**.<br/> <img src={useBaseUrl('img/traces/open-service-map.png')} alt="open servicemap" width="225"/>
+<img src={useBaseUrl('img/traces/open-service-map.png')} alt="open servicemap" width="400"/>
 
-The Service Map provides the following features:
+## Services List view
 
-* Zoom in and out on the map.
-* Hover over a service to see its connections to other services and the last 15 minutes of activity in terms of latency, requests, and errors.
-* Click on a service to open the Entity Inspector with the ability to drill down to traces, metrics, and the service dashboard.
-* Filter by application, if your tracing data has the `application=[app-name]` tag.
-* Search for a service by name.<br/> ![service map april 2021.png](/img/traces/service-map-example.png)
-* Color of services: The color of services is based on automatic (default) or manual thresholds you can configure in the settings menu. Red indicates an anomaly and blue indicates normal activity.
-* Shape of services: Remote services, like databases or external calls, automatically detected in client traffic, even without actual direct instrumentation, are visualized using the cloud icons instead of circles.<br/> <img src={useBaseUrl('img/traces/service-circles.png')} alt="servicecircles" width="250"/>
-* Arrows: The Service Map shows activity between services and their presence from the last 72 hours, however, if the service or connection was inactive for more than one hour, it is displayed with dashed lines.
-* Size of services: The size of services on the map is based on how active they are, where large circles are more active compared to small circles that are less active.  
+The **Services List** view provides a compact, high-level overview of your application service health insights and KPIs.
 
-## Settings
+![panel-service.png](/img/traces/service-list.png)
```

Comment:
Cant find that image, not sure why. On my local yarn instance this shows as outdated with "Service Map" title.

Reply:
You'll have to pull in the latest branch updates (`git checkout service-list`, then `git pull origin service-list`)